lib/translation-assistant.js

```diff
@@ -355,9 +355,17 @@ function createTemplate(englishStr, translatedStr, lang) {
     }
 }
 
-// This array is used both in translateMath and normalizeTranslatedMath
+// TODO: Need to extract this out of tests and use MATH_LOCALES instead
 var THOUSAND_SEPARATOR_AS_THIN_SPACE_LOCALES = ['cs', 'fr', 'de', 'pt-pt', 'nb', 'bg', 'pl', 'ro', 'nl', 'az', 'sv', 'it', 'hu', 'uk'];
 
+var MATH_LOCALES = {
```

Comment:
Is the intent behind `MATH_LOCALES` is to ease exporting these rules to the tests, and to communicate that they're related concepts?

If so, I suggest renaming to something like `MATH_LOCALES_RULES`, and upper case the the properties to indicate that they're constant:

```
const MATH_LOCALES_RULES = {
    NO_THOUSAND_SEP: ['ko', 'ps'],
    ...
};
```

Reply:
Okay, renaming definitely make sense, thanks. I wasn't sure about using upper case in keys, will do.

The most important reason is that these are used in another function: `normalizeTranslatedMath` so these needs to be separated. And at this point it prob. make sense to do it to all locale lists, even though not all of them are needed in that second function.
And, yes, this will make testing easier.